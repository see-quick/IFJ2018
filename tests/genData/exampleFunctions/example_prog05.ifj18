# Program 4: count cube volume


square_side = 0
area = 0

   print ("\nEnter the whole side of cube\x20\n")
   square_side = inputi
   #calculation of the area
   area = square_side * square_side *square_side

   print("\nVolume of the cube\x20", area, "\n")
