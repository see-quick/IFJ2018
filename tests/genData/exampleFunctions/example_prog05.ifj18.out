
Enter the whole side of cube 
5
Volume of the cube 125
